supadantic/models.py

```
@@ -105,7 +105,7 @@ def save(self: _M) -> _M:
         data returned from the database after the save operation.
 
         Returns:
-            (_M): The saved model instance, updated with data from the database (e.g.,
+            (_Model): The saved model instance, updated with data from the database (e.g.,
```

why `_Model`?

---

it complained that `_M` is too short... Initially I tought to change `_M` to `_Model` yet then decided to suppress the error since it does not make a lot of sense. WDYT?

---

@darya-shynkevich but it seems that you change it only in docstring, we can change it but everywhere